pkg/disttask/importinto/subtask_executor.go

```diff
@@ -295,10 +296,12 @@ func rebaseAllocatorBases(ctx context.Context, taskMeta *TaskMeta, subtaskMeta *
 		return errors.Trace(err)
 	}
 	etcd.SetEtcdCliByNamespace(etcdCli, keyspace.MakeKeyspaceEtcdNamespace(kvStore.GetCodec()))
-	r := autoIDRequirement{store: kvStore, etcdCli: etcdCli}
+	autoidCli := autoid.NewClientDiscover(etcdCli)
+	r := autoIDRequirement{store: kvStore, autoidCli: autoidCli}
```

Comment:
Maybe it is more efficient to reuse `etcdCli` and `autoidCli` globally. We can improve this in another PR. /cc @D3Hunter